airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -0,0 +1,129 @@
+from airflow.models import BaseOperator
+from jinja2 import Template
+from subprocess import PIPE
+import logging
+import re
+import subprocess
+import time
+import uuid
+
+YAML_TEMPLATE = """
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ job_name }}
+spec:
+  template:
+    spec:
+      containers: {% for container in containers %}
+      - name: {{ container.name }}
+        image: {{ container.image }}
+        command: {{ container.command }} {% endfor %}
+      restartPolicy: Never
+  backoffLimit: 0
+"""
+
+
+class KubernetesContainerInformation():
+    def __init__(self,
+                 name,
+                 image,
+                 command):
+        self.name = name
+        self.image = image
+        self.command = command
+
+
+class KubernetesJobOperator(BaseOperator):
```

Comment:
we should add some tests to this!

Reply:
one for the polling for job statuses may be useful. another for the yaml creation.

Reply:
tests are good! I will add some after I'm done working with the other comments/refactoring